toolz/functoolz.py

```diff
@@ -541,3 +541,32 @@ def do(func, x):
     """
     func(x)
     return x
+
+
+@curry
+def flip(func, a, b):
+    """Call the function call with the arguments flipped.
+
+    Parameters
+    ----------
+    f : callable
+        A binary function.
+    a : any
+        The second argument to `f`.
+    b : any
+        The first argument to `f`.
+
+    Returns
```

Comment:
Yeah, that should be enough.

And I understand--defaulting to numpy-style documentation is a good habit to have.